### ISSUE TITLE
working and maintained alternative to o2 anyone?

### ISSUE THREAD

**Issue by whysthatso** [Automattic/o2#173]
i'm looking for a working alternative for either the o2 plugin or the p2/o2 theme. is anybody here aware of one?

i think the basic functionality of a frontend edit/post box with link prefetching a la facebook is really underrated, and i hope maybe somebody has come across that, or could post here their strategies how to implement such a functionality via a plugin or a theme.

**Comment by lancewillett** [comment]
Thank you @whysthatso 

Might be worth raising this on the WordPress.org forums, too.

There are likely a few "Front End Editing" plugins out there as standalone services.

Of course, we welcome contributions here yet cannot promise quick fixes or attention.

**Comment by whysthatso** [comment]
> Thank you @whysthatso
> 
> Might be worth raising this on the WordPress.org forums, too.
> 
> There are likely a few "Front End Editing" plugins out there as standalone services.
> 
> Of course, we welcome contributions here yet cannot promise quick fixes or attention.

yep, i've put it [there](https://wordpress.org/support/topic/working-and-maintained-alternative-to-o2-plugin-p2-theme-anyone) now. wasn't too straight forward to find a good place. thanks for the tip. i also asked for support in the subreddit /r/wordpress

**Comment by mrjarbenne** [comment]
@whysthatso you might want to check out https://github.com/Automattic/liveblog which has a front end post editor. I'm not sure if it supports the Open Graph tags that would prefetch URL content, but that might give you a good start.

**Comment by yduke** [comment]
A wordpress theme named "Keep Minutes" might do the work:
https://keepmins.com/purchase-en/

**Comment by whysthatso** [comment]
> A wordpress theme named "Keep Minutes" might do the work:
> https://keepmins.com/purchase-en/

i did come across your offering very recently, and it does looking intriguing. however, the visualization of links (like facebook does it, an image, using the opengraph data, etc, like https://wordpress.org/plugins/visual-link-preview/) and adding wordpress categories on a per post basis, that would be a minimum requirement.

**Comment by yduke** [comment]
> > A wordpress theme named "Keep Minutes" might do the work:
> > https://keepmins.com/purchase-en/
> 
> i did come across your offering very recently, and it does looking intriguing. however, the visualization of links (like facebook does it, an image, using the opengraph data, etc, like https://wordpress.org/plugins/visual-link-preview/) and adding wordpress categories on a per post basis, that would be a minimum requirement.

Thanks for advices. But...
1 If there is a stable plugin for that, I will not build that function in Wordpress theme.
2 Categories are great, but tags are even better, especially when you are running a multi-user site, see twitter.
